solutions/azure-hub-spoke/azurepipeline.yml

```diff
@@ -34,6 +34,7 @@ pr:
       - /solutions/azure-hub-spoke/*
     exclude:
       - '/solutions/azure-hub-spoke/README.md'
+      - '/solutions/azure-hub-spoke/bicep/*'
```

Comment:
Did you want to remove 
![image](https://user-images.githubusercontent.com/671075/115455582-c59f4980-a1e7-11eb-8c37-07f29f1864fa.png)
at the same time since it's not covered in PR or merge to master?

Reply:
I don't think so, both native ARM and Bicep are still present, this exclusion is to prevent a build on the ARM template when only the Bicep tempalte has been modified.

Perhaps I do not understand the comment?